OxfordBodleian/Juel-Jensen/BDLaethe37.xml

```diff
@@ -66,13 +71,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                     </dimensions>
                                     <note corresp="#leafdim">Data on leaves dimensions taken from <locus target="#24r"/>.</note>
                                 </extent>
-                                
+
                                 <collation>
                                     <list>
                                         <item xml:id="q1">
                                             <dim unit="leaf">2</dim>
                                             <locus from="1r" to="96v"/>
-                                            <!--these should probably belong to the binding-->
+                                            <!--these should probably belong to the binding.
+                                            EDS (10.02.2022): I am not sure wether these leaves can be considered as part of a quire
+                                            to me they seems to be single leaves and I would not put them in the bindingDesc....in the case they do belong to a quire,
+                                            they should be considered as the outer bifolium, with all the other quires nested inside. Does the present encoding rapresent this structure?
```

Comment:
Picture JJ-11.0104.jpg is a bit blurred but it seems to me that fols 1 and 96 belong to an outer bifolium which covers the spine.

Reply:
Ah, you are right! Now I see it. I thought it was a white piece of leather but it's parchment. So, shall we keep them in the collation? They could be considered as extensions of a sort of spine lining...so be included in the bindingDesc

Reply:
I also think (as I did two years ago, apparently) that they should be described in the bindingDesc, they are not part of the quire structure as far as I understand it.

Reply:
So, q2 becomes q1 (and all the other quires change accordingly) and f. 1 and f. 96 go in the bindingDesc, right?

Reply:
I think so, as a consequence of the previous interpretation of this outer bifolium as part of the binding. But the best think would be to consult a handbook of codicology and see how such cases have been treated and if they have been considered in the collation.

Reply:
@MassimoVilla  I think it is a good idea, just I don't know where to look...there is not such an handbook ,unfortunately!...the only other similar manuscript I have seen is the ms 119 of the Casamari abbey in Veroli. When I encoded it for Manus online, I have not considered the parchment extensions from the spine in the collation. But the should be in the forthcoming catalogue...how did you encode it?
Do you know of other similar cases?

Reply:
Unfortunately MS Casamari MNC-018 (= 119) did not belong to my share of manuscripts and has not been described yet. As far as can be useful, this is what we wrote down: the textblock is wrapped in a leaf sewn to the guardleaves by means of parchment strips.

Reply:
I see, I see...no problem. I propose then a tentative solution that exclude f.1 and f. 96 from the collation. We can then evaluate if it's ok